blocks/hero/hero.css

```diff
@@ -1,37 +1,231 @@
-.hero-container .hero-wrapper {
-  max-width: unset;
-  padding: 0;
+main > .section.hero-container {
+    padding-top: 0;
+    padding-bottom: 0;
+    margin: 0;
+}
+
+@media (width < 960px) {
+    main > .section.hero-container .hero-wrapper {
+       padding: 0;
+    }
 }
 
-.hero {
-  position: relative;
-  padding: 40px 24px;
-  min-height: 300px;
+.hero-container .hero-wrapper {
+    max-width: unset;
+    padding: 0;
 }
 
 .hero h1 {
-  max-width: 1200px;
-  margin-left: auto;
-  margin-right: auto;
-  color: var(--background-color);
+    margin-bottom: var(--qui-spacing-0);
+}
+
+.hero.container {
+    padding: var(--qui-spacing-0);
+}
+
+.hero .hero-image .desktop-image {
+    display: none;
+}
+
+@media (width < 600px) {
```

Comment:
I have a couple of questions.

1. Should we only target media queries such as `width >= value` and handle the screens smaller than `value` under the "default" (without media queries) styles?
2. Should we group the styles this way? Styling followed by a media query or should we have all styles and at the end of the file the media queries grouped by rules for all styles/classes etc.?